relayer/middleware/index.ts

```diff
@@ -1,9 +1,9 @@
 export * from "./logger.middleware";
 export * from "./metrics.middleware";
-export * from "./missedVaas";
 export * from "./providers.middleware";
 export * from "./source-tx.middleware";
 export * from "./staging-area.middleware";
 export * from "./tokenBridge.middleware";
 export * from "./wallet";
+export * from "./missedVaasV3";
```

Comment:
not sure if this version of missed vaas classifies as middleware, should take it to a different folder?

Reply:
hmmm, great catch! I'll fix that